NOTICE-binary

```diff
@@ -66,15 +78,9 @@ Copyright 2017-2020 The Apache Software Foundation
 Apache Ratis Thirdparty Miscellaneous
 Copyright 2017-2020 The Apache Software Foundation
 
-Apache Spark
-Copyright 2014 and onwards The Apache Software Foundation
-
 ResolverUtil.java
 Copyright 2005-2006 Tim Fennell
 
-Apache Flink
-Copyright 2014-2022 The Apache Software Foundation
```

Comment:
it's a mistake, we neither copied code from Flink nor ship Flink binary artifacts in the binary release